extensions/vscode/src/utils/files.ts

```diff
@@ -314,9 +314,11 @@ export function vscodeOpenFiles(): string[] {
   window.tabGroups.all.forEach((tabGroup) => {
     tabGroup.tabs.forEach((tab) => {
       const input = tab.input as TabInputText;
-      const filePath = relativePath(input.uri);
-      if (filePath) {
-        openFileList.push(filePath);
+      if (input) {
+        const filePath = relativePath(input.uri);
+        if (filePath) {
+          openFileList.push(filePath);
+        }
       }
     });
```

Comment:
This isn't directly related to the specific change here, which is good, but when hitting this error in person it made me think: shouldn't the inside of this for(each) loop be in a try/catch such that a failure on any individual tabs doesn't mean that other, just tabs which are just fine will still be presented?

Reply:
With this change, we have achieved the same effect, since none of the inner function calls are expected to throw exceptions. Of course, that is only known by code browsing, vs. having language support for it like Java does.

I'm not currently confident that we have always taken the approach of degrading functionality upon error, vs. reporting a fatal error. I'm hopeful that we can look towards improving this in issue #2069.